src/target/at32f43x.c

```diff
@@ -89,7 +89,14 @@ static bool at32f43_mass_erase(target_s *target);
 #define AT32F43x_2K_OB_COUNT 256U
 #define AT32F43x_4K_OB_COUNT 2048U
 
-#define DBGMCU_IDCODE 0xe0042000U
+#define AT32F43x_DEBUG            0xe0042000U
+#define DBGMCU_IDCODE             (AT32F43x_DEBUG + 0U)
+#define AT32F43x_DEBUG_CTRL       (AT32F43x_DEBUG + 4U)
+#define AT32F43x_DEBUG_APB1_PAUSE (AT32F43x_DEBUG + 8U)
```

Comment:
Similar to the first comment, please prefix all of these with `AT32F43x_` and call the base address `AT32F43x_DBGMCU_BASE`, using the peripheral name as a secondary suffix for all of these. This is part of a drive to try and unify nomenclature across the code base and be more consistent about this.

Reply:
Renamed everything DEBUG-related to DBGMCU. Added a short doc block.